README.md

```diff
@@ -25,7 +25,18 @@ cmake ..
 make -j 4
 ```
 CMake tries to find the libraries and enables the corresponding make targets.
-After make have finished you can run e.g. `./gearshifft_cufft`.
+After `make` have finished you can run e.g. `./gearshifft_cufft`.
+
+## Install
+
+Set `CMAKE_INSTALL_PREFIX` and `GEARSHIFFT_INSTALL_CONFIG_PATH` as you wish, otherwise defaults are used.
+```
+mkdir release && cd release
+cmake -DCMAKE_INSTALL_PREFIX=/home/user/gearshifft 
```

Comment:
@tdd11235813 if you write `$HOME/gearshifft` and append a single `\` and the end of the line this even works with copy & paste